test/publicExports.js

```diff
@@ -1,10 +1,10 @@
-var Ganache = require(process.env.TEST_BUILD
+const assert = require("assert");
```

Comment:
Why'd you rename this file?

Reply:
Thought we agreed on camelCase.

Reply:
I opened an issue for all occurrences in test files. https://github.com/trufflesuite/ganache-core/issues/353